cmd/controller/run.go

```diff
@@ -73,12 +77,14 @@ func Run(opts Options, runLog logr.Logger) {
 	}
 
 	{ // add controller for apps
+		appRefTracker := reftracker.NewAppRefTracker()
```

Comment:
instead of using & in following uses of this var, let's have NewAppRefTracker return a pointer to AppRefTracker

Reply:
Added